OpenRA.m

```diff
@@ -121,15 +121,20 @@ - (void)launchGameWithArgs: (NSArray *)gameArgs
 
 	launched = YES;
 
-	// Default values - can be overriden by setting MonoMinVersion and MonoGameExe in Info.plist
+	// Default values - can be overriden by setting certain keys Info.plist
 	NSString *gameName = @"OpenRA.Game.exe";
+	NSString *defaultModId = @"modchooser";
```

Comment:
Hardcoding this to modchooser isn't great considering that we're about to kill that.  Can you please default this to `nil`, and then only add the launch arguments if it is non-nil?